specification/iotcentral/data-plane/Microsoft.IoTCentral/stable/2022-07-31/iotcentral.json

```diff
@@ -0,0 +1,6113 @@
+            },
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      }
+    }
+  },
+  "parameters": {
+    "BaseDomain": {
+      "in": "path",
+      "name": "baseDomain",
+      "description": "The base domain for all Azure IoT Central service requests.",
+      "x-ms-parameter-location": "client",
+      "type": "string",
+      "x-ms-skip-url-encoding": true,
+      "default": "azureiotcentral.com",
+      "required": true
+    },
+    "CommandName": {
+      "in": "path",
+      "name": "commandName",
+      "description": "Name of this device command.",
+      "x-ms-parameter-location": "method",
+      "type": "string",
+      "required": true
+    },
+    "ComponentName": {
+      "in": "path",
+      "name": "componentName",
+      "description": "Name of the device component.",
+      "x-ms-parameter-location": "method",
+      "type": "string",
+      "required": true
+    },
+    "DeviceGroupId": {
+      "in": "path",
+      "name": "deviceGroupId",
+      "description": "Unique ID for the device group.",
+      "x-ms-parameter-location": "method",
+      "type": "string",
+      "maxLength": 255,
+      "pattern": "^[a-zA-Z0-9_-]*$",
+      "required": true
+    },
+    "DeviceId": {
+      "in": "path",
+      "name": "deviceId",
+      "description": "Unique ID of the device.",
+      "x-ms-parameter-location": "method",
+      "type": "string",
+      "required": true
+    },
+    "DeviceTemplateId": {
+      "in": "path",
+      "name": "deviceTemplateId",
+      "description": "Unique [Digital Twin Model Identifier](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/dtdlv2.md#digital-twin-model-identifier) of the device template.",
+      "x-ms-parameter-location": "method",
+      "type": "string",
+      "required": true
+    },
+    "EnrollmentGroupId": {
+      "in": "path",
+      "name": "enrollmentGroupId",
+      "description": "Unique ID of the enrollment group.",
+      "x-ms-parameter-location": "method",
+      "type": "string",
+      "maxLength": 255,
+      "pattern": "^[a-zA-Z0-9-]*$",
+      "required": true
+    },
+    "Entry": {
+      "in": "path",
+      "name": "entry",
+      "description": "Entry of certificate only support primary and secondary.",
+      "x-ms-parameter-location": "method",
+      "type": "string",
+      "enum": [
+        "primary",
+        "secondary"
+      ],
+      "x-ms-enum": {
+        "name": "CertificateEntry",
+        "modelAsString": false,
+        "values": [
+          {
+            "value": "primary",
+            "name": "Primary",
+            "description": "The entry of primary certificate"
+          },
+          {
+            "value": "secondary",
+            "name": "Secondary",
+            "description": "The entry of secondary certificate"
+          }
+        ]
+      },
+      "required": true
+    },
+    "Filter": {
+      "in": "query",
+      "name": "$filter",
```

Comment:
Don't use "$" prefix on parameter names.

Reply:
Removed $filter, $top, $orderby from July GA, will revise this and get preview on Sep.